compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -251,6 +251,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
       {
       self()->setUseSSEForSinglePrecision();
       self()->setUseSSEForDoublePrecision();
+      self()->setSupportsAutoSIMD();
```

Comment:
I'd rather that it was commented out or just not there rather than #if 0 - ideally just not there since it can be added when SIMD support is added to the code generator later.